### PR TITLE
Issue 1 - Initial implementation

### PR DESCRIPTION
**Pull request by rodfersou**

### CODE REVIEW

File: src/collective/behavior/automaticrelateditems/profiles/default/metadata.xml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 <?xml version="1.0"?>
 <metadata>
   <version>1</version>
+  <dependencies>
```
Comment (hvelarde) [inline src/collective/behavior/automaticrelateditems/profiles/default/metadata.xml]: you don't need to install those packages here; they are just dependencies.

File: src/collective/behavior/automaticrelateditems/subscriber.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+def handler(context, event):
+    """Automatically create relation items by tags
+
+    :param context: the item being processsed
+    :type context: Dexterity-based content type
+    :param event: event rised
+    :type event: subscriber event
+    """
+    if len(context.relatedItems) > 0:
```
Comment (hvelarde) [inline src/collective/behavior/automaticrelateditems/subscriber.py]: this will raise `AttributeError` if the context doesn't provide the behavior.

**Comment by rodfersou** [comment]
@hvelarde should I remove the subscriber after [this commit](https://github.com/simplesconsultoria/collective.behavior.automaticrelateditems/pull/2/commits/7d512fcc20230bac30b5d857efd40f4d57330b90) ?

**Comment by hvelarde** [comment]
@rodfersou you'll be able to answer yourself after you update the requirements and the implementation details included on the README file; currently what we have here is different from what @agnogueira discussed with you yesterday.

**Comment by rodfersou** [comment]
@hvelarde I'm asking for directions on how this package should behave.. but no problem, thank you for your answer

**Comment by rodfersou** [comment]
@hvelarde ok, thank you

**Comment by rodfersou** [comment]
@hvelarde ready to review

**Comment by hvelarde** [comment]
we'll do it on Monday

**Pull request merged**